vendor/licenses/grammar/wren-sublime.txt

```diff
@@ -0,0 +1,28 @@
+---
+type: grammar
+name: wren-sublime
+license: other
+---
```

Comment:
Can you please update your fork with the latest master, and try replacing the grammar with itself using:

```
script/add-grammar --replace wren-sublime https://github.com/munificent/wren-sublime
```

This should hopefully use the new Licensed code to correctly and automatically detect the license.